src/main/resources/db/migration/V8_43__add_columns_to_parameter_table.sql

```diff
@@ -0,0 +1,41 @@
+ALTER TABLE parameter
```

Comment:
This script could be a bit more defensive. If this crashes mid-execution it could possibily be a bit of a headache

Reply:
@MickeWallner Any thoughts about this?

Reply:
Any suggestions how it could be done more defensive? @cheezi0747 I think we can live with only columns are added. Then modifying data is in a transaction.

Reply:
if this script fails during the modification you wont be able to re-run the script as is. Adding `if exists` and `if not exists` could prevent this.
e.g:
```sql
ALTER TABLE if exists parameter
  ADD COLUMN if not exists parameter_order INTEGER NOT NULL DEFAULT 0;
  ```